<launch>
    <arg name="left_arm_port" default="5555"/>
    <arg name="right_arm_port" default="5556"/>

    <!-- Left Arm Grasping Service -->
    <group ns="L_panda">
        <node name="grasp_client" pkg="dual_arm_control" type="combined_client_srv.py" output="screen" respawn="true">
            <param name="arm_id" value="L_panda"/>
            <param name="port" value="$(arg left_arm_port)"/>
        </node>
    </group>

    <!-- Right Arm Grasping Service -->
    <group ns="R_panda">
        <node name="grasp_client" pkg="dual_arm_control" type="combined_client_srv.py" output="screen" respawn="true">
            <param name="arm_id" value="R_panda"/>
            <param name="port" value="$(arg right_arm_port)"/>
        </node>
    </group>
</launch>
